File: JavaScript_Courses/JavaScript_Basics/JS_BP_2022_10/03_Conditional_Statements_Advanced_Exercise/02_Summer_Outfit.js
function summerOutfit(input) {

    let degrees = Number(input[0]);
    let timeOfDay = input[1];

    let outfit = "";
    let shoes = "";


    if (timeOfDay == "Morning") {
        if (degrees >= 10 && degrees <= 18) {
            outfit = "Sweatshirt";
            shoes = "Sneakers";
        } else if (degrees > 18 && degrees <= 24) {
            outfit = "Shirt";
            shoes = "Moccasins";
        } else if (degrees >= 25) {
            outfit = "T-Shirt";
            shoes = "Sandals";
        }
    } else if (timeOfDay == "Afternoon") {
        if (degrees >= 10 && degrees <= 18) {
            outfit = "Shirt";
            shoes = "Moccasins";
        } else if (degrees > 18 && degrees <= 24) {
            outfit = "T-Shirt";
            shoes = "Sandals";
        } else if (degrees >= 25) {
            outfit = "Swim Suit";
            shoes = "Barefoot";
        }
    } else if (timeOfDay == "Evening") {
        if (degrees >= 10 && degrees <= 18) {
            outfit = "Shirt";
            shoes = "Moccasins";
        } else if (degrees > 18 && degrees <= 24) {
            outfit = "Shirt";
            shoes = "Moccasins";
        } else if (degrees >= 25) {
            outfit = "Shirt";
            shoes = "Moccasins";
        }
    }

    console.log(`It's ${degrees} degrees, get your ${outfit} and ${shoes}.`);
}


// summerOutfit(["28", "Evening"]);


// if (degrees >= 10 && degrees <= 18) {
//     if (timeOfDay == "Morning") {
//         outfit = "Shirt";
//         shoes = "Moccasins";
//     } else if (timeOfDay == "Afternoon") {
//         outfit = "Shirt";
//         shoes = "Moccasins";
//     } else if (timeOfDay == "Evening") {
//         outfit = "Shirt";
//         shoes = "Moccasins";
//     }
 
// } else if (degrees > 18 && degrees <= 24) {
//     if (timeOfDay == "Morning") {
//         outfit = "Shirt";
//         shoes = "Moccasins";
//     } else if (timeOfDay == "Afternoon") {
//         outfit = "Shirt";
//         shoes = "Moccasins";
//     } else if (timeOfDay == "Evening") {
//         outfit = "Shirt";
//         shoes = "Moccasins";
//     }
// } else if (degrees >= 25) {
//     if (timeOfDay == "Morning") {
//         outfit = "Shirt";
//         shoes = "Moccasins";
//     } else if (timeOfDay == "Afternoon") {
//         outfit = "Shirt";
//         shoes = "Moccasins";
//     } else if (timeOfDay == "Evening") {
//         outfit = "Shirt";
//         shoes = "Moccasins";
//     }
// }

File: Python_Courses/Python_Basics/Python_PB_2025_03/03_Conditional_Statements_Advanced_Exercise/02_Summer_Outfit.py
degrees = int(input())
time_of_day = input()  # "Morning", "Afternoon", or "Evening"

outfit = ''
shoes = ''
if time_of_day == 'Morning':
    if 10 <= degrees <= 18:  # 10 <= градуси <= 18
        outfit = 'Sweatshirt'
        shoes = 'Sneakers'
    elif 18 < degrees <= 24:  # 18 < градуси <= 24
        outfit = 'Shirt'
        shoes = 'Moccasins'
    elif degrees >= 25:  # градуси >= 25
        outfit = 'T-Shirt'
        shoes = 'Sandals'

elif time_of_day == 'Afternoon':
    if 10 <= degrees <= 18:  # 10 <= градуси <= 18
        outfit = 'Shirt'
        shoes = 'Moccasins'
    elif 18 < degrees <= 24:  # 18 < градуси <= 24
        outfit = 'T-Shirt'
        shoes = 'Sandals'
    elif degrees >= 25:  # градуси >= 25
        outfit = 'Swim Suit'
        shoes = 'Barefoot'

elif time_of_day == 'Evening':  # else:
    if 10 <= degrees <= 18:  # 10 <= градуси <= 18
        outfit = 'Shirt'
        shoes = 'Moccasins'
    elif 18 < degrees <= 24:  # 18 < градуси <= 24
        outfit = 'Shirt'
        shoes = 'Moccasins'
    elif degrees >= 25:  # градуси >= 25
        outfit = 'Shirt'
        shoes = 'Moccasins'

print(f'It\'s {degrees} degrees, get your {outfit} and {shoes}.')
